scripts/publish.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
Maybe add some instructions about this on the Readme?

Also let's confirm if the script works fine on both Linux and Mac before merging

Reply:
Added a short line about it, I don't know what more to write since it doesn't take any arguments or such.